src/vue/modules/dividend-form/index.vue

```diff
@@ -0,0 +1,530 @@
+                    {{ holder.available | formatMoney }}
+                    {{ form.ownedAsset.code }}
+                  </td>
+                  <td v-if="form.amount">
+                    {{ getDividendAmount(holder) | formatMoney }}
+                    {{ form.asset.code }}
+                  </td>
+                  <td v-else>
+                    &mdash;
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <template v-if="!balanceHolders.length && isSignersLoaded">
+              <p class="dividend__fee-holders-not-found">
+                {{ 'dividend-form.asset-holders-not-found' | globalize }}
+              </p>
+            </template>
+          </div>
+
+          <div class="app__form-actions dividend__action">
+            <button
+              v-ripple
+              v-if="!formMixin.isConfirmationShown"
+              type="submit"
+              class="app__button-raised"
+              :disabled="formMixin.isDisabled || !balanceHolders.length"
+              form="dividend-form"
+            >
+              {{ 'dividend-form.dividend-btn' | globalize }}
+            </button>
+            <form-confirmation
+              v-if="formMixin.isConfirmationShown"
+              @ok="hideConfirmation() || submit()"
+              @cancel="hideConfirmation()"
+            />
+          </div>
+        </form>
+      </template>
+      <template v-else>
+        <h2 class="app__page-heading">
+          {{ 'dividend-form.no-assets-heading' | globalize }}
+        </h2>
+        <p class="app__page-explanations app__page-explanations--secondary">
+          {{ 'dividend-form.no-assets' | globalize }}
+        </p>
+        <router-link
+          to="/tokens"
+          tag="button"
+          class="app__button-raised dividend__action"
+        >
+          {{ 'dividend-form.discover-assets-btn' | globalize }}
+        </router-link>
+      </template>
+    </template>
+    <template v-else-if="isFailed">
+      {{ 'dividend-form.can-not-load-assets' | globalize }}
+    </template>
+    <template v-else>
+      <loader message-id="dividend-form.loading-msg" />
+    </template>
+  </div>
+</template>
+
+<script>
+import debounce from 'lodash/debounce'
+import FormMixin from '@/vue/mixins/form.mixin'
+import FormConfirmation from '@/vue/common/FormConfirmation'
+import Loader from '@/vue/common/Loader'
+
+import { mapActions, mapGetters, mapMutations } from 'vuex'
+import { Bus } from '@/js/helpers/event-bus'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import EmailGetter from '@/vue/common/EmailGetter'
+import { types } from './store/types'
+
+import { FEE_TYPES, PAYMENT_FEE_SUBTYPES, Wallet } from '@tokend/js-sdk'
+import { api, initApi } from './_api'
+import { AssetRecord } from './wrappers/asset.record'
+import required from 'vuelidate/src/validators/required'
+import { amount, maxDecimalDigitsCount, noMoreThanAvailableOnBalance } from '../../../validators'
```

Comment:
Here are more than 80 symbols in the line. Do you have an ESLint issue about it? If not, config your IDE correctly